Header header
string time_update
string measurement_update
string simulation
string filename
string bloom_pub #bloom location publisher to gazebo world

#filename ==> file name in the folder.no path is needed